web/src/js/__tests__/ducks/_tflow.js

```diff
@@ -8,7 +8,7 @@ export default function(){
         "alpn_proto_negotiated": "http/1.1",
         "cipher_name": "cipher",
         "clientcert": null,
-        "id": "75bfd3cd-a084-4d84-a063-b0804dc91342",
+        "id": "4a18d1a0-50a1-48dd-9aa6-d45d74282939",
```

Comment:
OK, this commit is helpful in unintended ways. We need to set the ids and timestamps to constant values before exporting, otherwise this file will always be changed.